static/lang/en.json

```diff
@@ -2653,6 +2653,15 @@
                 "Hint": "Settings to exclude packs from loading",
                 "Name": "Compendium Browser Packs"
             },
+            "CompendiumBrowserSources": {
+                "AllowEmptySources": "Allow entries with empty sources to be displayed.",
+                "AllowUnknownSources": "Allow entries with unknown sources to be displayed.",
+                "DeleteAllInfo": "This can not be undone, but you may reconfigure this. The settings will be set to allow all sources, which include known sources, empty sources and unknown sources.",
+                "DeleteAllQuestion": "Are you sure you want to delete the allowed sources settings?",
+                "DeleteAllTitle": "Delete allowed sources settings",
+                "Hint": "Settings to exclude sources from loading",
+                "Name": "Compendium Browser Allowed Sources"
+            },
```

Comment:
To make it clear that this controls what players can see, can the language here be changed to make that explicit? E.g., "Player Access" instead of "Compendium Browser Allowed Sources"

Reply:
Currently it also filters for the GM, so "Player Access" may not be perfect. I will think about a better name.

Reply:
@stwlam I did two things:

1. I added a setting to allow GMs to ignore filtering, which is also now the default.
2. I changed and added new strings on the UI, see the updated screenshot in description.

In the end I used "Player Access", because I did not find a better name for this, but I just used it for the list of sources.

Reply:
It's fine to not treat this as a permission-related feature for players, but the language in the labels shouldn't include terms  like "allowed" or "permitted" in that case.

Reply:
@stwlam I changed the labels, I hope this is better. I also did some refactoring of the code to remove "allowed" in variable names, to avoid confusion.

Also I updated / added screenshots to show an example for the equipment screen, before and after applying the settings for the filtering + the new settings section.